tower-grpc-examples/src/routeguide/client.rs

```diff
@@ -55,18 +60,31 @@ pub fn main() {
             RouteGuide::new(conn)
         })
         .and_then(|mut client| {
-            client
```

Comment:
You can probably keep this with the additional calls?

Reply:
Redone preserving the original `GetFeature` call.